lib/dbservice/sessions/session.ex

```diff
@@ -60,22 +61,24 @@ defmodule Dbservice.Sessions.Session do
       :platform_id,
       :type,
       :auth_type,
-      :activate_signup,
+      :signup_form,
       :id_generation,
       :redirection,
       :pop_up_form,
-      :number_of_fields_in_pop_form,
-      :form_schema_id
+      :popup_form_id,
+      :signup_form_id
     ])
     |> validate_required([
       :name,
+      :session_id,
```

Comment:
i dont think session_id should be part of. validation.. when we create sessions on session-creator, initially there is no session id. later on, a lambda function creates session id and updates db rows